default/stringtables/en.txt

```diff
@@ -6646,27 +6646,37 @@ GREETINGS_INTRO
 Greetings Esteemed Entity, and Welcome to the FreeOrion Universe!
 
 GREETINGS_GUIDE_REFERENCE
-Newcomers to this universe may particularly wish to consult the [[encyclopedia QUICK_START_GUIDE_TITLE]].
+To help you understand how these three areas of government - ships, planets, and research - are managed in your empire, please have a look at the [[encyclopedia QUICK_START_GUIDE_TITLE]] which we have prepared for you.
```

Comment:
If you really want to keep this general change, it needs substantial rewriting. Avoid second person reference to the player ("you", "your") in general, and remove all the overly specific story info like "elected Emperor" and "new-founded". Basically, in-game text does not address the player as a person, but rather states information.

Reply:
rgr wilco